tinygrad/runtime/ops_metal.py

```diff
@@ -1,15 +1,64 @@
 from __future__ import annotations
 import os, subprocess, pathlib, ctypes, tempfile, functools
-import Metal, libdispatch
-from typing import List, Any, Tuple, Optional
-from tinygrad.helpers import prod, getenv, DEBUG, unwrap2
+from typing import List, Any, Tuple, Optional, cast, TypeVar
+from tinygrad.helpers import prod, getenv, DEBUG
 from tinygrad.device import Compiled, Compiler, CompileError, LRUAllocator
 from tinygrad.renderer.cstyle import MetalRenderer
+from ctypes.macholib.dyld import dyld_find
```

Comment:
Why do you need this? Just `/System` isn't universal?

Reply:
i thought dyld_find is safer, but just read that /System is the core part of the OS so it should be universal